agent/lib/kontena/workers/stats_worker.rb

```diff
@@ -34,9 +34,9 @@ def on_node_info(topic, info)
     end
 
     def start
-      info 'waiting for cadvisor'
+      debug 'waiting for cadvisor'
       sleep 1 until cadvisor_running?
```

Comment:
Not necessary for this PR, but these debug+sleep should be refactored to `wait(message: "waiting for cadvisor to be running") { cadvisor_running? }`